### DISCUSSION TITLE
配置参数这块不能设置学习率吗？

### DISCUSSION THREAD

**Discussion by chenchen333-dev** [zai-org/ChatGLM3#1099] [Ideas]
最新的微调项目文件，在配置参数这块不能设置学习率吗？

**Comment by zRzRzRzRzRzRzR** [comment]
有呀

**Comment by chenchen333-dev** [comment]
> 有呀

你好！执行微调训练命令后，能成功加载模型，且打印了首条训练数据处理信息，之后运行到下面的这些信息，也就是加载完分词器模型时就结束了，还没开始训练把，这是怎么回事呢？
from the default dictionary ...
Building prefix dict from the default dictionary ...
Building prefix dict from the default dictionary ...
Building prefix dict from the default dictionary ...
Dumping model to file cache /tmp/jieba.cache
Dumping model to file cache /tmp/jieba.cache
Dumping model to file cache /tmp/jieba.cache
Dumping model to file cache /tmp/jieba.cache
Loading model cost 0.902 seconds.
Prefix dict has been built successfully.
Loading model cost 0.924 seconds.
Prefix dict has been built successfully.
Loading model cost 0.940 seconds.
Prefix dict has been built successfully.
Loading model cost 0.958 seconds.
Prefix dict has been built successfully.

100%|██████████| 35/35 [17:06<00:00, 29.32s/it]
I0410 17:57:49.461486 4611 ProcessGroupNCCL.cpp:837] [Rank 2] NCCL watchdog thread terminated normally
I0410 17:57:49.461515 4616 ProcessGroupNCCL.cpp:837] [Rank 3] NCCL watchdog thread terminated normally
I0410 17:57:49.489074 4614 ProcessGroupNCCL.cpp:837] [Rank 0] NCCL watchdog thread terminated normally
I0410 17:57:49.494719 4610 ProcessGroupNCCL.cpp:837] [Rank 1] NCCL watchdog thread terminated normally

**Reply by zRzRzRzRzRzRzR** [reply]
这这是报错了吧，没有正常开始训练

**Reply by chenchen333-dev** [reply]
没有报ERROR，成功加载完分词器模型就停止了，没有开始训练，这是最后一条信息：100%|██████████| 35/35 [17:06<00:00, 29.32s/it]
I0410 17:57:49.461486 4611 ProcessGroupNCCL.cpp:837] [Rank 2] NCCL watchdog thread terminated normally
I0410 17:57:49.461515 4616 ProcessGroupNCCL.cpp:837] [Rank 3] NCCL watchdog thread terminated normally
I0410 17:57:49.489074 4614 ProcessGroupNCCL.cpp:837] [Rank 0] NCCL watchdog thread terminated normally
I0410 17:57:49.494719 4610 ProcessGroupNCCL.cpp:837] [Rank 1] NCCL watchdog thread terminated normally
